typescript/ecs/ecs-network-load-balanced-service/index.ts

```diff
@@ -26,8 +31,9 @@ class BonjourECS extends cdk.Stack {
       }
     });
 
-    // Output the DNS where you can access your service
-    new cdk.CfnOutput(this, 'LoadBalancerDNS', { value: ecsService.loadBalancer.loadBalancerDnsName });
```

Comment:
Why are you removing this?

Reply:
It will contain duplicated value. Since the higher level construct already output the same.